tgui/packages/tgui/interfaces/ParticleAccelerator.js

```diff
@@ -42,7 +81,158 @@ export const ParticleAccelerator = (props, context) => {
             </LabeledList.Item>
           </LabeledList>
         </Section>
+        {assembled ? (
+          ''
+        ) : (
+          <Section
+            title={
+              orientation
+                ? 'EM Acceleration Chamber Orientation: ' + capitalize(orientation)
+                : 'Place EM Acceleration Chamber Next To Console'
+            }
+          >
+            {orientation === 0 ? (
+              ''
+            ) : orientation === 'north' || orientation === 'south' ? (
+              <LayoutVertical />
+            ) : (
+              <LayoutHorizontal />
+            )}
+          </Section>
+        )}
       </Window.Content>
     </Window>
   );
 };
+
+const LayoutHorizontal = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { assembled, power, strength, max_strength, icon, layout_1, layout_2, layout_3, orientation } = data;
+  return (
+    <Table>
+      <TableRow width="40px">
+        {(orientation === 'east' ? layout_1 : layout_3).slice().map((item) => (
+          <Table.Cell key={item.name}>
+            <Tooltip content={`${item.name} Status:${item.status}, Direction:${dir2text(item.dir)}`}>
```

Comment:
```suggestion
            <Tooltip content={`${item.name} Status: ${item.status}, Direction: ${dir2text(item.dir)}`}>
```

Reply:
The lack of spaces there is to keep category and value on the same line. I tried separating the lines manually with \n so I could put spaces there but it didn't work.